java-persistence/src/main/java/com/whitecloak/training/inventory/service/ItemService.java

```diff
@@ -1,9 +1,23 @@
 package com.whitecloak.training.inventory.service;
 
 import com.whitecloak.training.inventory.controller.request.ItemForm;
+import com.whitecloak.training.inventory.controller.response.ItemPaginationResponse;
 import com.whitecloak.training.inventory.controller.response.ItemResource;
+import com.whitecloak.training.inventory.persistence.entity.ItemEntity;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+
+import java.util.List;
 
 public interface ItemService {
 
     ItemResource create(ItemForm form);
+    ItemEntity getItem(Long id);
+//    Page<ItemEntity> findAllByPage(Pageable pageable);
```

Comment:
Remember dead code anti-pattern?

Leaving unused comments in your code is one way to do it.